Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamerBase.h

```diff
@@ -269,6 +269,10 @@ class MediaPlayerPrivateGStreamerBase : public MediaPlayerPrivateInterface
     Condition m_drawCondition;
     Lock m_drawMutex;
 #endif
+#if ENABLE(LEGACY_ENCRYPTED_MEDIA_V1) && USE(OPENCDM)
```

Comment:
If you honor my comment below, you won't need to move this, just make the getter function protected.